src/utils/dataFormat.go

```diff
@@ -65,6 +65,19 @@ func RoundValues(num1, num2 float64) ([]float64, string) {
 		units = " per trillion "
 	}
 
+	if inBytes {
```

Comment:
Can a few unit tests be written for this?

Reply:
Yea this can be done. Will do along with CPU percent calculations